src/components/Web3Status/index.tsx

```diff
@@ -205,11 +210,13 @@ function Web3StatusInner() {
 
 export default function Web3Status() {
   return (
-    <PrefetchBalancesWrapper>
-      <Web3StatusInner />
-      <Portal>
-        <PortfolioDrawer />
-      </Portal>
-    </PrefetchBalancesWrapper>
+    <div data-testid="web3-status">
+      <PrefetchBalancesWrapper>
+        <Web3StatusInner />
+        <Portal>
+          <PortfolioDrawer />
+        </Portal>
+      </PrefetchBalancesWrapper>
+    </div>
```

Comment:
this reads like you're adding the div just so it can be used by a test to select the data-testid--can you find a way to test w/o modifying the app like this?